src/oc/web/components/comments.cljs

```diff
@@ -150,6 +156,15 @@
                                     :empty (zero? (count sorted-comments))})}
           (if (pos? (count sorted-comments))
             [:div.comments-internal-scroll
+             {:on-scroll (fn [e]
+                           (let [comments-internal-scroll (js/$ ".comments-internal-scroll")]
+                             ;; when a user scrolls up,
+                             ;; turn off the scroll to bottom after render
+                             (when (and (> (.data comments-internal-scroll "lastScrollTop")
+                                           (.scrollTop comments-internal-scroll))
+                                        @(::scroll-bottom-after-render s))
+                               (reset! (::scroll-bottom-after-render s) false))
+                             (.data comments-internal-scroll "lastScrollTop" (.scrollTop comments-internal-scroll))))}
```

Comment:
Maybe set `(.scrollTop comments-internal-scroll)` in a var in the let above to avoid it changing value btw the two uses?
Not sure if this is subject to race condition though, your call

Reply:
It should be the case that the user scrolls and then `data` value is set. This value only changes when the user scrolls again. Since there is only one way to change that value I don't think it should be a race condition.

Am I missing something?

Reply:
nope, makes sense.